src/components/header/Header.jsx

```diff
@@ -32,6 +40,30 @@ export default class Header extends React.Component {
             userWidget = (
                 <UserMenu user={ userData }/>
             );
+
+            let callAssignments = this.props.assignments.getIn(['assignmentList', 'items']);
+
+            if (callAssignments.size > 0) {
+                let callUrl = '//call.' + process.env.ZETKIN_DOMAIN + '/';
+
+                callLink = (
+                    <Button href={  callUrl } labelMsg="header.call"
+                            className="Header-navLink linkCall"/>
+                );
+            }
+
+            let membership = this.props.orgs.getIn(['membershipList', 'items']);
+
+            let isOfficial = membership.filter(item => item.get('role') != null);
```

Comment:
This is not a boolean, as the name `isOfficial` seems to indicate. Either convert it to a boolean (e.g. using `find()` instead of `filter()`, see below) or rename the variable to something like `officialMemberships`.

This is how it could be given a boolean value (in which case the check on line 59 needs to change):
```
let isOfficial = !!membership.find(item => item.get('role) != null);

if (isOfficial) //...
```